packages/unit-testing/stubs/action/ActionInstance.js

```diff
@@ -0,0 +1,71 @@
+define([], function () {
+    /**
+     * @param {Object} options
+     * @param {string=} options.recordId record instance ID
+     */
+    function Action(options) {
+
```

Comment:
Missing properties: description, id, label, parameters and recordType.

Can be checked at https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_1510761537.html#bridgehead_1509379229

As far as I have seen properties can be added as we have done alreadu on https.ClientResponse

        /**
         * Response code.
         * @name ClientResponse#code
         * @type number
         * @readonly
         * @throws {SuiteScriptError} READ_ONLY_PROPERTY when setting the property is attempted
         */        
        this.code = undefined;